src/components/ImageDetails/PhotoDetails.vue

```diff
@@ -84,7 +84,7 @@
           :href="image.foreign_landing_url"
           target="_blank"
           rel="noopener"
-          class="button is-success margin-vertical-normal"
+          class="button is-success margin-vertical-normal margin-horizontal-small"
```

Comment:
The issue says the margin is between the button and the reuse survey component, so a vertical margin. I think you should change this to `margin-bottom-small` and remove the horizontal margins

Reply:
Done